spinalcordtoolbox/scripts/sct_check_dependencies.py

```diff
@@ -157,7 +157,7 @@ def add_bash_profile(string):
 
 def get_dependencies(requirements_txt=None):
     if requirements_txt is None:
-        requirements_txt = sct_dir_local_path("requirements.txt")
+        requirements_txt = sct_dir_local_path("generated_requirements.txt")
```

Comment:
Oh, of course! Good catch here. I didn't even consider that `sct_check_dependencies` relies on `requirements.txt`. 

This actually seems like it fixes a bug, I think? Right now, when `sct_check_dependencies` is run for stable releases, it's only checking against the flexible requirements. Now, with your new changes, it will check against the stricter frozen requirements, which is what we want!

Thank you for this!